### DISCUSSION TITLE
Projects Board Automation Not Working As Expected

### DISCUSSION THREAD

**Discussion by d3ssy** [community/community#23389] [Projects and Issues]
<p>When adding automation to project board columns for issues and PR, I have set it up so that</p>
<ol>
<li>Newly added or reopened issues go into <strong>ToDo</strong> column</li>
<li>Closed issues go into <strong>Done</strong> column</li>
<li>Newly added PR’s go into <strong>Review in progress</strong> column</li>
<li>(As expected, I move issues from <strong>ToDo</strong> to <strong>In progress</strong> manually)</li>
</ol>
<p>All of the above work. However, when merging PR created with a linked issue (either via keyword <strong>closes <span class="hashtag">#ISSUE_NUMBER</span></strong> or manually linking the issue), the PR will automatically move from <strong>Review in progress</strong> column to <strong>Done</strong> column but the issue itself does not move from <strong>InProgress</strong> column to <strong>Done</strong> column, and is not marked as <strong>Closed</strong>.</p>
<p>Is this a known bug?</p>

**Comment by tajmone** [comment, marked as answer]
<aside class="quote no-group" data-username="d3ssy" data-post="1" data-topic="188980">
<div class="title">
<div class="quote-controls"></div>
 d3ssy:</div>
<blockquote>
<p>Is this a known bug?</p>
</blockquote>
</aside>
<p>My general experience is that it works as expected.</p>
<p>Beware that closing Issues via auto-linked references like <code>closes #&lt;PR_NUM&gt;</code> only take effect when merging into the default branch (i.e. <code>main</code>/ <code>master</code>, unless you’ve changed it). So, if you’re merging in a dev branch, the Issue won’t close until it’s also merged into the default branch.</p>
<p>If that’s the case (i.e. if you adopt a dev-branch based workflow), you might consider setting the baseline dev branch as the default branch.</p>
